std_msgs/Header header

# Position Error
geometry_msgs/Point pos_error
# Velocity Error
geometry_msgs/Point vel_error